### PR TITLE
Add DispatcherTimer to the list of available timer classes

### PR DESCRIPTION
**Pull request by softworkz**
This is just meant to be a hint.
Why is there's no feedback section at the bottom?

Feel free to close this and do it in a better way.

### CODE REVIEW

File: xml/System.Threading/Timer.xml [hunk 1]
```diff
@@ -80,12 +80,13 @@ Sub TimerCallback(state As Object)
...
 > - <xref:System.Windows.Forms.Timer?displayProperty=nameWithType> (.NET Framework only), a Windows Forms component that fires an event and executes the code in one or more event sinks at regular intervals. The component has no user interface and is designed for use in a single-threaded environment; it executes on the UI thread.
 > - <xref:System.Web.UI.Timer?displayProperty=nameWithType> (.NET Framework only), an ASP.NET component that performs asynchronous or synchronous web page postbacks at a regular interval.
+> - <xref:System.Windows.Threading.DispatcherTimer?displayProperty=nameWithType> A timer that is integrated into the Dispatcher queue which is processed at a specified interval of time and at a specified priority.
```
Comment (gewarren) [inline xml/System.Threading/Timer.xml:89]:
```suggestion
> - <xref:System.Windows.Threading.DispatcherTimer?displayProperty=nameWithType>, a timer that's integrated into the `Dispatcher` queue. This timer is processed with a specified priority at a specified time interval.
```

File: xml/System.Threading/Timer.xml [hunk 2]
```diff
@@ -80,12 +80,13 @@ Sub TimerCallback(state As Object)
  The timer delegate is specified when the timer is constructed, and cannot be changed. The method does not execute on the thread that created the timer; it executes on a <xref:System.Threading.ThreadPool> thread supplied by the system.  
   
 > [!TIP]
-> .NET includes four classes named `Timer`, each of which offers different functionality:  
+> .NET includes five classes named `Timer`, each of which offers different functionality:
```
Comment (gewarren) [inline xml/System.Threading/Timer.xml:83]:
```suggestion
> .NET includes several timer classes, each of which offers different functionality:  
```

**Comment by softworkz** [comment]
> The feedback section was removed from the API ref docs simply because we could not answer every issue in a timely fashion.

That's unfortunate, I really liked it. Often just for reading the comments from other users..

I understand that there's a small edge for this getting misused as a support channel, but still - most of the questions I read - were still justified to be covered by the documentation to avoid those coming up as support questions..

**Comment by gewarren** [comment]
@kouvel Do you approve this PR?

**Comment by kouvel** [comment]
Makes sense to me but I don't have much background or context in that area. @stephentoub, who would know more about this?

**Comment by stephentoub** [comment]
I see that Windows Forms' counterpart is included already, so it seems fine to list WPF's as well.

**Comment by softworkz** [comment]
It was just that I had momentarily forgotten the name of `DispatcherTimer` and this "list of .NET timer classes" didn't help as it didn't include it.

But if I had known that it would allocate so many people's time, I wouldn't have submitted. Apologies..

**Comment by KalleOlaviNiemitalo** [comment]
There is a similar list of timer classes at [System.Timers.Timer](https://docs.microsoft.com/dotnet/api/system.timers.timer?view=net-5.0#remarks). It would be good to keep these synchronized: <https://github.com/dotnet/dotnet-api-docs/blob/50c98f5d147a5d8f6a96fcbd455b3cdee29d8643/xml/System.Timers/Timer.xml#L85-L91>

That list seems to be out of date already, as System.Windows.Forms.Timer is available on .NET Core 3.0 and later.

I suppose the "this topic" self-reference prevents [\!INCLUDE](https://docs.microsoft.com/en-us/contribute/markdown-reference#included-markdown-files) from being used.

**Comment by gewarren** [comment]
@KalleOlaviNiemitalo Good point. Would you like to create a new PR for System.Timers.Timer?

**Comment by KalleOlaviNiemitalo** [comment]
No, I wouldn't.

**Pull request merged**